requirements/tools.txt

```diff
@@ -18,6 +18,7 @@ MarkupSafe==1.0
 mccabe==0.6.1
 numpy==1.12.1
 packaging==16.8
+pip==9.0.1
```

Comment:
Randomly flailing through Google found https://github.com/pyupio/pyup/pull/169 which suggests that it *does* issue PRs for this sort of thing.